lint/lint.go

```diff
@@ -132,6 +133,9 @@ func (d *Dashboard) GetPanels() []Panel {
 	for _, row := range d.Rows {
 		p = append(p, row.Panels...)
 	}
+	for _, panel := range d.Panels {
+		p = append(p, panel.Panels...)
+	}
```

Comment:
Sounds like this should really be recursive? Adding any sub-panels in rows or panels?

Reply:
I thought about it, but I don't know any case today where you would have more than one level of nesting. I can make it recursive anyway, you tell me :)

Reply:
Yeah I'd go for it just in case.  But then I like recursion ;-)

Reply:
Fine :). I have updated the PR with a recursive version

Reply:
(Sorry to be nit picky)